backend/src/main/scala/cromwell/backend/validation/ValidatedRuntimeAttributesBuilder.scala

```diff
@@ -68,14 +69,12 @@ trait ValidatedRuntimeAttributesBuilder {
     val errorsOrValuesMap: Seq[(String, ErrorOr[Any])] =
       validationsForValues.map(validation => validation.key -> validation.validate(values))
 
-    import scalaz.Scalaz._
-
-    val emptyResult: ErrorOr[List[(String, Any)]] = List.empty[(String, Any)].success
+    val emptyResult: ErrorOr[List[(String, Any)]] = List.empty[(String, Any)].validNel
     val validationResult = errorsOrValuesMap.foldLeft(emptyResult) { (agg, errorOrValue) =>
-      agg +++ {
+      agg |+| {
```

Comment:
ToL: More a question-- We have to implement this empty plus semigroup combine ourselves? I would have guessed that there would have been a default monoid implemented for validation. (aka fought for literally hours to try and get the syntax/implicits right)